Corriger les erreurs de formatage bloquantes pour suivre les instructions du README

En suivant le README littéralement on obtient actuellement l'erreur suivante:

```
openfisca_france/model/prestations/minima_sociaux/aah.py:284:33: W504 line break after binary operator
openfisca_france/model/prestations/minima_sociaux/aah.py:285:103: W504 line break after binary operator
openfisca_france/model/prestations/minima_sociaux/aah.py:355:58: W504 line break after binary operator
openfisca_france/model/prestations/minima_sociaux/aah.py:356:43: W504 line break after binary operator
openfisca_france/model/prestations/minima_sociaux/aah.py:357:35: W504 line break after binary operator
Makefile:37: recipe for target 'check-style' failed
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -79,7 +79,7 @@ jobs:
       - run:
           name: Lint files
           command: |
-            make check-syntax-errors
+            make check-syntax-errors check-style
             .circleci/lint-changed-python-files.sh
 
   check_version_and_changelog:

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,13 @@
 # Changelog
 
+### 47.0.1 [#1349](https://github.com/openfisca/openfisca-france/pull/1349)
+
+* Amélioration technique.
+* Périodes concernées : toutes.
+* Zones impactées : `aah.py`.
+* Détails :
+  - Corrige des erreurs de formatage bloquantes pour suivre les instructions du README
+
 # 47.0.0 [#1343](https://github.com/openfisca/openfisca-france/pull/1343)
 
 * Évolution du système socio-fiscal.

--- a/openfisca_france/model/prestations/minima_sociaux/aah.py
+++ b/openfisca_france/model/prestations/minima_sociaux/aah.py
@@ -281,9 +281,9 @@ class aah_plafond_ressources(Variable):
         en_couple = individu.famille('en_couple', period)
         af_nbenf = individu.famille('af_nbenf', period)
         montant_max = law.minima_sociaux.aah.montant
-        return montant_max * (1 +
-                              en_couple * law.minima_sociaux.aah.majoration_du_plafond_pour_un_couple +
-                              law.minima_sociaux.aah.tx_plaf_supp * af_nbenf)
+        return montant_max * (1
+                            + en_couple * law.minima_sociaux.aah.majoration_du_plafond_pour_un_couple
+                            + law.minima_sociaux.aah.tx_plaf_supp * af_nbenf)
 
 
 class aah_base(Variable):
@@ -352,10 +352,10 @@ class eligibilite_caah(Variable):
         locataire_foyer = (individu.menage('statut_occupation_logement', period) == TypesStatutOccupationLogement.locataire_foyer)
         salaire_net = individu('salaire_net', annee_precedente, options=[ADD])
 
-        return ((taux_incapacite >= taux_incapacite_min) *
-            ((aah > 0) | (benef_asi > 0)) *
-            not_(locataire_foyer) *
-            (salaire_net == 0)
+        return ((taux_incapacite >= taux_incapacite_min)
+            * ((aah > 0) | (benef_asi > 0))
+            * not_(locataire_foyer)
+            * (salaire_net == 0)
             )
 
 

--- a/setup.py
+++ b/setup.py
@@ -7,7 +7,7 @@ from setuptools import setup, find_packages
 
 setup(
     name = "OpenFisca-France",
-    version = "47.0.0",
+    version = "47.0.1",
     author = "OpenFisca Team",
     author_email = "contact@openfisca.fr",
     classifiers = [

